js/formstone.js

```diff
@@ -10,6 +10,9 @@ Drupal.behaviors.formstone.attach = function(context, settings) {
     if (navigator.userAgent.match(/Android (\d+)/i)) {
       $('.selecter').css('pointer-events', 'none');
     }
+
+    // unset selecter for select2 fields
+    $('.webform-component-autocomplete select', context).selecter('destroy');
```

Comment:
would be better to fix the line 6:
```js
var $selects = $('.webform-component:not(.webform-component-autocomplete) select', context).selecter();
```

should prevent initiating in the first place...